.github/workflows/tests.yml

```diff
@@ -24,5 +24,9 @@ jobs:
         python -m pip install --upgrade pip
         pip install tox tox-gh-actions
 
+    - name: Check Environment
```

Comment:
If this test failed, will the action lastest-dependencies still take effect?

Reply:
Yes, the action latest-dependencies is responsible for creating a PR whenever installing SigPro results in newer dependencies. The PR then runs the usual tests which will catch any issues with the latest dependencies and the library. If this check failed, the PR would still be created but the test would appear as failing indicating we need to go in and fix something. 

However, it's very unlikely this test would ever fail since a clean `pip` install should be successfully resolving package dependencies in the same way the latest dependency checker is.